From: Anthony Sottile <asottile@umich.edu>
Date: Tue, 30 Jun 2020 09:22:48 -0700
Subject: Revert "bpo-40874 Update the required libmpdec version for the
 decimal module (GH-21202) "

This reverts commit 119de0eba839993cf6a909dba5d60202ad5566d6.
---
 Modules/_decimal/_decimal.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Modules/_decimal/_decimal.c b/Modules/_decimal/_decimal.c
index fb4e020..7061729 100644
--- a/Modules/_decimal/_decimal.c
+++ b/Modules/_decimal/_decimal.c
@@ -36,8 +36,8 @@
 #include "docstrings.h"
 
 
-#if !defined(MPD_VERSION_HEX) || MPD_VERSION_HEX < 0x02050000
-  #error "libmpdec version >= 2.5.0 required"
+#if !defined(MPD_VERSION_HEX) || MPD_VERSION_HEX < 0x02040100
+  #error "libmpdec version >= 2.4.1 required"
 #endif
 
 
